src/index.js

```diff
@@ -14,7 +15,16 @@ module.exports = function (input) {
     }
 
     console.log('[babel-plugin-testable]', ...logs);
-  }
+  };
+
+  const isTestEnvironment = () => {
+    const isTestEnv = (process.env.NODE_ENV === 'test'
+      || process.env.BABEL_ENV === 'test'
+      || typeof global.it === 'function');
+
+    log('isTestEnvironment', isTestEnv);
+    return isTestEnv;
```

Comment:
You moved this code up? I would suggest grabbing the `.git` repo via npm in a project and testing it locally to ensure nothing broke with the changes.

Reply:
Yeah, it was an eslint issue since it was being used before its defined